### PR TITLE
V3 changes

### PR DESCRIPTION
**Pull request by ThiagoRRamos**
Making the Client work with the most recent version (2017-03-29)

### CODE REVIEW

File: shippo/test/test_rate.py [hunk 1]
```diff
@@ -44,18 +44,6 @@ def test_retrieve(self):
     def test_invalid_retrieve(self):
         self.assertRaises(shippo.error.APIError, shippo.Rate.retrieve, 'EXAMPLE_OF_INVALID_ID')
 
-    @shippo_vcr.use_cassette(cassette_library_dir='shippo/test/fixtures/rate')
```
Comment (davidhtien) [inline shippo/test/test_rate.py]: We don't need any tests like this any more?

File: shippo/test/integration/test_integration.py [hunk 2]
```diff
@@ -62,17 +62,6 @@ def test_list_accessors(self):
         address['foo'] = 'bar'
         self.assertEqual(address.foo, 'bar')
 
-    def test_raise(self):
```
Comment (davidhtien) [inline shippo/test/integration/test_integration.py]: Don't need this test anymore?

Comment (ThiagoRRamos) [inline shippo/test/integration/test_integration.py]: We are not raising exceptions on those cases anymore. Just marking them as imcomplete.

This test was really redundant to one in `test_address`. So I removed it

File: shippo/test/test_rate.py (hunk 1 above)
Comment (ThiagoRRamos) [inline shippo/test/test_rate.py]: We are not supporting listing all rates anymore

**Pull request merged**